example_bt2/params copy.yaml

```diff
@@ -0,0 +1,4 @@
+:type: map
```

Comment:
What are they using in their example, string or array of string?

Reply:
They use array of strings, but we do not support [that] (https://github.com/TApplencourt/metababel/blob/45878ae43e653a53764a7b5b401678821fb98d97/lib/metababel/bt2_values_generator.rb#L11), so I used string.